src_c/mixer.c

```diff
@@ -34,6 +34,14 @@
 
 #define PyBUF_HAS_FLAG(f, F) (((f) & (F)) == (F))
 
+#define CHECK_CHUNK_VALID(CHUNK, RET)                                      \
+    if ((CHUNK) == NULL) {                                                 \
+        PyErr_SetString(PyExc_RuntimeError,                                \
+                        "__init__() was not called on Sound object so it " \
+                        "failed to setup correctly.");                     \
+        return (RET);                                                      \
+    }
+
```

Comment:
Nit: I think this should have used the `do { ... } while(0)` trick.  However, since we do not use this with `if (...) CHECK_CHUNK_VALID(...); else ...` then I do not think it matters and I think having this merged sooner is a bigger net-benefit then pedantic correctness of a macro.

Reply:
Ya I feel it isn't completely necessary. Unless someone else chimes in